TextRPG_Team25/Battle.cs

```diff
@@ -40,7 +41,52 @@ private void PlayerAttack(Monster selected)
                 Console.WriteLine($"HP {currentHp}-> HP {selected.HP}");
             }
 
-            
+  
         }
+
+        private void SpawnEnemy() // 몬스터 소환
+        {
+            Random rand = new Random();
+            int spawnNum = rand.Next(1, 4);
+            RanMonsterSpawn(spawnNum);
+        }
+
+        private void RanMonsterSpawn(int spawnNum) // 랜덤 몬스터 소환
```

Comment:
책임 분리는 잘 하신 것 같습니다!!
이 생성함수는 랜덤 몬스터를 고르고 반환하는 것까지 하고, 이 몬스터를 배틀에 참여시키기 위한 행동은 위에 만드셨던 SpawnEnemy쪽에서 맡으면 깔끔할 것 같네여! 요 함수 자체는 Monster.cs로 이동하겠습니다. 전체 로직은 좋다고 생각합니다 👍🏻


```suggestion
        public static Monster GenerateRandomMonster()
{
    var random = new Random();
    int index = random.Next(monsters.Count); // 리스트 내부 속성이므로 .Count 사용했습니다
    return monsters[index];
}
```